### ISSUE TITLE
AT LEAST 25 TROJAN VIRUSES ATTACHED TO YOUR WINE ENGINES

### ISSUE THREAD

**Issue by nicebub** [Gcenx/WineskinServer#48]
I downloaded and installed the WS9Wine5.7 and my antivirus quickly came back showing 25 files as having trojans.  I'd bet you have more as well attached to other engines and files.

**Comment by Gcenx** [comment]
Your antivirus is giving false positives you can check Valves Proton GitHub issues for confirmation also see the threads on Winehq about this, you can also find information on this on CodeWeavers own site explaining the issues.

Pleas note;\
WS9 Engines are Winehq portable releases that are repackaged into Wineskin Engines on your system, I only provide WineCX Engines for direct download via Winery.

See #26 where I’ve also explained the above and shown the difference between two compiled wine binaries one using stock mingw-binutils and the other using the Proton patched version.

TLDR;\
Your anti virus thinks a wine file is a virus because it looks like a Windows file.

**Comment by nicebub** [comment]
Why wouldn’t this happen with standard wine skin. I’ve never had this issue
before in the past.

**Comment by Gcenx** [comment]
That’s simple, older versions of wine didn’t provide PE binaries where recent versions of wine does.

Since around Wine-4.8 mingw-w64 has become required to cross-compile wine components.\
As your lightly not aware mingw-64 is a cross-compiling toolchain allows you to compile Windows binaries on no Windows systems.

Wine now provides functional replacements for real Windows binaries, now most anti-virus like for example Norton Endpoint Security see the file let’s say regedit.exe and notice is a PE binary but not a known release from Microsoft so it’s auto flagged.

Now much older versions of wine that were not compiled in this manner, instead wine provided fake Windows binaries, lets usr the above example regedit.exe is just loads regedit.so.

The problem with using none PE binaries is some software expects a real PE binary a fake one doesn’t suffice.

**Comment by twardoch** [comment]
This is worth adding to a FAQ in the main README — I just ran into the same problem.
